contracts/utils/AddressString.sol

```diff
@@ -36,7 +36,7 @@ library AddressToString {
         stringBytes[0] = '0';
         stringBytes[1] = 'x';
 
-        for (uint256 i; i < length; ++i) {
+        for (uint256 i = 0; i < length; ++i) {
```

Comment:
In a previous code arena they told us this is less efficient, and they are equivalent otherwise. I haven't tested it tho.

Reply:
ok, slither warning, but will revert.